tests/unittests/benchmark/task/test_task_wrapper.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Dict, List
+
+import pytest
+from orion.benchmark.task.base import BaseTask
+from orion.benchmark.task.task_wrapper import FixTaskDimensionsWrapper, TaskWrapper
+from orion.core.io.space_builder import SpaceBuilder
+
+
+class DumbTask(BaseTask):
+    def __init__(self, max_trials: int, seed: int = None, a: int = 100, b: int = 10, c: int = 1):
+        super().__init__(max_trials=max_trials)
+        self.a = a
+        self.b = b
+        self.c = c
+        self.seed = seed
+
+    def call(self, x: Dict) -> List[Dict]:
+        y = self.a * x["a"] + self.b * x["b"] + self.c * x["c"]
+        return [dict(name="dumb_task", type="objective", value=y)]
+
+    def get_search_space(self) -> Dict[str, str]:
+        return {
+            "a": "uniform(0, 10, discrete=True)",
```

Comment:
It's good to use different priors just to make sure to detect any mix-up bug latter on.

Reply:
Are there ready-made fixtures I can use for testing stuff like this?

Reply:
No, but using different bounds for a b and c will do the trick.

Reply:
They at least have a different maximum value now.